features/admin.feature

```diff
@@ -222,6 +225,14 @@ Feature:  Administration
     And I should not see "Delete"
     And I should see "search.gov"
 
+    When I follow "Sitemaps"
+    Then I should see "search.gov/sitemap.xml"
+    And I follow "Delete" and confirm "Are you sure you want to delete this sitemap?"
```

Comment:
After this line, I would add:
`Then I should not see "search.gov/sitemap.xml"`
to confirm the sitemap was deleted.